ui/packages/catalog-ui-search/src/main/webapp/templates/geocoder.handlebars

```diff
@@ -11,9 +11,7 @@
  *
  **/
  --}}
+ {{!-- TODO DDF-4200 Revisit map loading forever when this is removed --}}
```

Comment:
It's causing issues because the other items in the toolbar have been appending themselves to this view's html.